magic_demo/pubspec.yaml

```diff
@@ -61,6 +61,8 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
+  
+  package_info_plus: ^3.1.2
```

Comment:
Curious if it would be a better patch to bump the version in the SDK than adding the dependency in the demo app